.openpublishing.redirection.json

```diff
@@ -1125,7 +1125,8 @@
     },
     {
       "source_path": "docs/csharp/programming-guide/concepts/linq/linq-to-xml-axes.md",
-      "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/linq-to-xml-axes-overview"
+      "redirect_url": "/dotnet/standard/linq/linq-xml-axes-overview"
+      "redirect_document_id": true
```

Comment:
given that this redirect seems to exist already, I don't think you should add redirect true here.

```suggestion
```